ui/app/store/store.js

```diff
@@ -15,4 +15,4 @@ module.exports = function configureStore (initialState) {
       thunkMiddleware,
     ),
   ))
-}
+};
```

Comment:
:eyes:

Reply:
The next few files have this as well

Reply:
I'll rebase this on develop, now that #7754 is merged—that should fail this

Reply:
Welp, it doesn't fail these 😢 I'll fix these manually